src/types.ts

```diff
@@ -162,9 +162,9 @@ export type InstallationAccessTokenData = {
   expiresAt: UTC_TIMESTAMP;
   permissions: Permissions;
   repositorySelection: REPOSITORY_SELECTION;
-  repositoryIds?: number[];
-  repositoryNames?: string[];
-  singleFileName?: string;
+  repositoryIds: number[] | undefined;
+  repositoryNames: string[] | undefined;
+  singleFileName?: string | undefined;
```

Comment:
This changes the meaning so that these fields are always required.

Currently the type makes it so that the field is not always present, and could be undefined.